editor/src/components/inspector/inspector-common.ts

```diff
@@ -404,6 +404,23 @@ export const nukeAllAbsolutePositioningPropsCommands = (
   ]
 }
 
+export const setAbsolutePositioningPropsCommands = (path: ElementPath): Array<CanvasCommand> => {
+  return [
+    addContainLayoutIfNeeded('always', path),
+    setProperty('always', path, PP.create('style', 'position'), 'absolute'),
+    setProperty('always', path, PP.create('style', 'left'), 'absolute'),
+    setProperty('always', path, PP.create('style', 'right'), 'absolute'),
+    setProperty('always', path, PP.create('style', 'top'), 'absolute'),
+    setProperty('always', path, PP.create('style', 'position'), 'absolute'),
+    deleteProperties('always', path, [
+      PP.create('style', 'position'),
+      PP.create('style', 'left'),
+      PP.create('style', 'right'),
+      PP.create('style', 'top'),
+      PP.create('style', 'bottom'),
+    ]),
```

Comment:
what on earth is this doing? :D

Reply:
😬 fixed in https://github.com/concrete-utopia/utopia/pull/3203/commits/af36fe9a693e86330467ab753252bcfea20177ab